.circleci/config.yml

```diff
@@ -22,6 +22,12 @@ jobs:
     executor: << parameters.jdk >>
     steps:
       - checkout
+      - run:
+          name: Sleep to avoid resource name conflict
+          command: |
+            if [[ $CIRCLE_JOB != "build-jdk8" ]]; then
+              sleep $((RANDOM % 30))
```

Comment:
How about using `sleep $((15 + RANDOM % 15))` to always sleep 15~30 seconds?
BTW, what is the purpose of having both jdk8 and jdk17 build?
Just for checking whether the project is still jdk8 compatible? I feel we can drop the support of jdk8 considering it has been deprecated.

Reply:
Thanks. I changed the sleep strategy.

In my understanding, JDK 8 is maintained by some vendors.
https://endoflife.date/eclipse-temurin / https://endoflife.date/redhat-build-of-openjdk